### DISCUSSION TITLE
endogenous priors

### DISCUSSION THREAD

**Discussion by pfjulio** [IRIS-Solutions-Team/IRIS-Toolbox#369] [General]
Hi,

Is there any "simple" way to introduce endogenous priors à lá Christiano, Trabandt and Walentin (2011) in estimation?

**Comment by jaromir-benes** [comment]
Haha, I'm pleased to inform you that the "endogenous" priors, that should
correctly be called "system priors", are actually my own and my colleague
Michal Andrle's invention :)) So yes, of course Iris has them, in an
extremely flexible way going beyond what you find in any academic paper. So
next time, please refer to them more properly :))))

You can impose priors on the following properties of the model, and you can
combine any number of these:

* any transformation of any point of the *autocovariance* function
* any transformation of any point of the *autocorrelation* function
* any transformation of any point of any kind of *shock responses*
* any transformation of any point of the *power spectrum* function
* any transformation of any point of the *spectral density* function
* any transformation of any point of any kind of *frequency response*
function
* any transformation of any *eigenvalue* number

See https://github.com/IRIS-Solutions-Team/Tutorial-System-Priors for some
examples and a couple of slides (in the doc folder)

Best
Jaromir
